frameworks/agno/travel-planner/agentuity_agents/travel_planner/travel_agent.py

```
@@ -0,0 +1,123 @@
+        {Detailed schedule with times and activities}
+
+        [Continue for each day...]
+
+        ## Budget Breakdown 💰
+        - Accommodation: {cost}
+        - Activities: {cost}
+        - Transportation: {cost}
+        - Food & Drinks: {cost}
+        - Miscellaneous: {cost}
+
+        ## Important Notes ℹ️
```

Ruff (0.13.1)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)